sorc/ncep_post.fd/FIXED.f

```diff
@@ -287,11 +296,12 @@ SUBROUTINE FIXED
 !$omp parallel do private(i,j)
          DO J=JSTA,JEND
            DO I=1,IM
+             GRID1(I,J) = SPVAL
              IF (MODELNAME == 'NMM') THEN
                IF( (abs(SM(I,J)-1.) < 1.0E-5) ) THEN
                  GRID1(I,J) = SST(I,J)
                ELSE
-                 GRID1(I,J) = THS(I,J)*(PINT(I,J,LM+1)/P1000)**CAPA
+                 IF(THS(I,J)<SPVAL) GRID1(I,J) = THS(I,J)*(PINT(I,J,LM+1)/P1000)**CAPA
```

Comment:
suggest changing to
IF(THS(I,J)<SPVAL .and. PINT(I,J,LM+1)<SPVAL) GRID1(I,J) = THS(I,J)*(PINT(I,J,LM+1)/P1000)**CAPA

Reply:
will do

Reply:
will do

Reply:
done